src/vcSettings.cpp

```diff
@@ -822,3 +827,24 @@ udResult vcSettings_UpdateLanguageOptions(vcSettings *pSetting)
 epilogue:
   return result;
 }
+
+void vcSettings_UpdateServerFlag(vcSettings *pSettings)
+{
+    UDCOMPILEASSERT(udLengthOf(g_vcMapTileServerName) == vcMTSF_Count, "Update array");
+
+    pSettings->maptiles.serverFlag = vcMTSF_Count;
+    for (size_t i = 0; i < vcMTSF_Count; i++)
```

Comment:
++i as per our coding standard

Reply:
Ok, thanks for your reminding, I will read our compilation standard carefully.